src/index.js

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './pages/App';
+// import App from './pages/App';
 import Charcuterie from 'pages/Charcuterie';
 import reportWebVitals from './utils/reportWebVitals';
+import TextIcon from 'components/TextIcon'
 
 ReactDOM.render(
   <React.StrictMode>
     {/* <App /> */}
     <Charcuterie >
       <p>Hi</p>
+      <TextIcon textLetter={'a'} size={'small'} color={'#8EDB31'}>small</TextIcon>
```

Comment:
do we wanna use constants for color?

Reply:
how would the colours for each user icon get picked (in the future when we start implementing that part)? will it be something like randomly generating from a list of colour constants and then passed into TextIcon?

Reply:
random colors from a set of constants most likely, but since this is charcuterie i'm cool with just a string literal for now